wsgi/odyssy/events/views.py

```diff
@@ -6,8 +6,7 @@
 
 def event(request):
     events_list = Event.get_all_events()
-    num_items = request.GET.get('num_items', default=10)
-    events = utils.paginate_view(request, events_list, num_items)
+    events, num_items = utils.paginate_view(request, events_list)
```

Comment:
@Prakash2403: Can you make this function return page number too? Just getting num_items is not sufficient in many cases, knowing the page number would help.